mozilla_vpn/explores/events.explore.lkml

```diff
@@ -2,8 +2,10 @@ include: "//looker-hub/mozilla_vpn/explores/events.explore.lkml"
 
 explore: +event_counts {
   sql_always_where:  ${events.submission_date} >= '2010-01-01'
-    AND (
-      ${events.metadata__header__x_telemetry_agent} LIKE "%Kotlin%"
-      OR ${events.metadata__header__x_telemetry_agent} LIKE "%JS%"
-    ) ;;
+      AND (
+        CASE WHEN (SAFE_CAST(SPLIT(${events.client_info__app_display_version}, '.')[SAFE_ORDINAL(1)] AS INTEGER) < 2 OR (SAFE_CAST(SPLIT(${events.client_info__app_display_version}, '.')[SAFE_ORDINAL(1)] AS INTEGER) = 2 AND SAFE_CAST(SPLIT(${events.client_info__app_display_version}, '.')[SAFE_ORDINAL(2)] AS INTEGER) < 16 ))
```

Comment:
It's hard to tell what this logic is doing at a glance (especially when it's all on one line like this), so I'd suggest having some explanatory comments like you had before.
```suggestion
        -- For version below 2.16 Kotlin and JS were the main Glean integrations.
        -- For version 2.16 and above Kotlin, Swift, and Rust are the main Glean integrations.
        CASE WHEN (SAFE_CAST(SPLIT(${events.client_info__app_display_version}, '.')[SAFE_ORDINAL(1)] AS INTEGER) < 2 OR (SAFE_CAST(SPLIT(${events.client_info__app_display_version}, '.')[SAFE_ORDINAL(1)] AS INTEGER) = 2 AND SAFE_CAST(SPLIT(${events.client_info__app_display_version}, '.')[SAFE_ORDINAL(2)] AS INTEGER) < 16 ))
```
(ditto for the other explores)